x-pack/plugins/apm/public/style/index.scss

```diff
@@ -0,0 +1,2 @@
+// Import the EUI global scope so we can use EUI constants
+@import 'src/legacy/ui/public/styles/_styling_constants';
```

Comment:
This import is loaded into _every_ sass file by default. You're loading this into your app, but it essentially just compiles down to an empty file. The imported file is just the variables and mixins that kibana uses.

You should be able to delete it, and your reference to it.

Reply:
Thanks, will do!